style.css

```diff
@@ -537,11 +595,25 @@ footer h3 {
   }
 
   .navbar-link {
-    font-size: 20px;
+    font-size: 17px;
     font-weight: 500;
     width: 20%;
   }
 
+  #company {
+    color: #ec5242;
+    padding: 2%;
+    border: 4px solid #ee4935;
+    white-space: nowrap;
+    margin-left: 8%;
+    text-decoration: none;
+    width: 20%;
+    display: flex;
+    justify-content: center;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+  }
```

Comment:
- [ ] (OPTIONAL) Kindly reduce the padding-bottom or margin-bottom of this button in the mobile view as it is too large 👍
- Your design:
![Annotation 2022-05-18 105745](https://user-images.githubusercontent.com/85113567/169016482-064b7990-b001-4857-be37-918535cc3f4c.jpg)

- Required design:
![Annotation 2022-05-18 105814](https://user-images.githubusercontent.com/85113567/169016542-ac389a6b-6222-4ba6-8f27-58f7600c7bd1.jpg)